Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/PersonDetailsViewModel.kt

```diff
@@ -116,7 +116,8 @@ class PersonDetailsViewModel @AssistedInject constructor(
 
         return UiState(
```

Comment:
Handle the logic in the viewmodel and send an gobackEvent if the corresponding numberofCertificates is reached